### PR TITLE
Mention ML more on homepage and advertisers page

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

File: ethicalads-theme/templates/ea/advertisers.html [hunk 1]
```diff
@@ -47,6 +47,7 @@ <h1 class="display-3 font-weight-light" data-aos="fade-left">Highly targeted ad
       <div class="col-md-6 mb-3">
         <h2 class="font-weight-bold mb-8">Reach engaged developers</h2>
         <p class="font-size-lg text-gray-800">When you run a campaign with us, your ads are prominently displayed on vital developer resources and tools, so your solution reaches real software developers.</p>
+        <p class="font-size-lg text-gray-800">Our machine learning audience classifier makes sure your ads are shown to the audience you are trying to reach whether that is data scientists, frontend developers, or security engineers.</p>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/advertisers.html:50]:
```suggestion
        <p class="font-size-lg text-gray-800">Our machine learning audience classifier makes sure your ads are shown to the audience:  data scientists, frontend developers, security engineers, and more.</p>
```

Tighten up copy a bit.

**Review by ericholscher — APPROVED** [review]
Looks good 👍

File: ethicalads-theme/templates/ea/advertisers.html (hunk 1 above)
Comment (davidfischer) [reply ethicalads-theme/templates/ea/advertisers.html:50]: To the "right" audience?

Comment (davidfischer) [reply ethicalads-theme/templates/ea/advertisers.html:50]: Or to "your" audience? To "the" audience feels odd.

**Pull request merged**